docker/Makefile

```diff
@@ -54,6 +54,15 @@ build-from-local-ubi8-artifacts: dockerfile env2yaml
 	  (docker kill $(HTTPD); false);
 	-docker kill $(HTTPD)
 
+build-from-local-observability-sre-artifacts: dockerfile
+	docker run --rm -d --name=$(HTTPD) \
+	           -p 8000:8000 --expose=8000 -v $(ARTIFACTS_DIR):/mnt \
+	           python:3 bash -c 'cd /mnt && python3 -m http.server'
+	timeout 120 bash -c 'until curl -s localhost:8000 > /dev/null; do sleep 1; done'
+	docker build --progress=plain --network=host -t $(IMAGE_TAG)-observability-sre:$(VERSION_TAG) -f $(ARTIFACTS_DIR)/Dockerfile-observability-sre data/logstash || \
+	  (docker kill $(HTTPD); false); \
+	-docker kill $(HTTPD)
```

Comment:
locally on my mac, my docker container does not shut down when this command is prefixed with `-`, which causes a subsequent invocation to fail since the container is in use.

Reply:
Ah. Paired with your other comment, this makes sense.

Reply:
yep! The `\` line continuation was resulting in the literal `-docker` command rather than using the `make ` syntax of prefixing a command you dont care if you get a non-zero exit code for with `-`.